tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -82,28 +84,91 @@ async def test_create_sensors(
     assert state.attributes.get(ATTR_STATE_CLASS) == SensorStateClass.TOTAL
     assert state.attributes.get(ATTR_DEVICE_CLASS) == SensorDeviceClass.ENERGY
 
+    state = hass.states.get("sensor.heat_meter_heat_usage")
+    assert not state
+
+    state = hass.states.get("sensor.heat_meter_heat_previous_year_gj")
+    assert state
+    assert state.state == "111.0"
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == UnitOfEnergy.GIGA_JOULE
+
+    state = hass.states.get("sensor.heat_meter_heat_previous_year")
+    assert not state
+
     state = hass.states.get("sensor.heat_meter_volume_usage")
     assert state
     assert state.state == "456.0"
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == UnitOfVolume.CUBIC_METERS
     assert state.attributes.get(ATTR_STATE_CLASS) == SensorStateClass.TOTAL
 
+    entity_reg = er.async_get(hass)
```

Comment:
Why did you remove the injected fixture?

Reply:
Did I? I don't think there was one in this test. There is the fixture you added to _test_config_flow.py_, but I don't think there was another one. Seems like a good idea though.

Reply:
According to github, you are adding this line inside the PR.
You shouldn't be adding the line.
```suggestion
```

Reply:
Ah yes, I was offset by the terminology, but I get it now. Thanks for making it explicit. Could be a rebase thing indeed. I've split a previous PR (which is much better, but did leave some left overs apparently.